ui/perfherder/compare/CompareTable.jsx

```diff
@@ -120,11 +118,11 @@ export default class CompareTable extends React.PureComponent {
             </th>
             <th className="table-width-lg">Base</th>
             {/* empty for less than/greater than data */}
-            <th className="table-width-sm" />
+            <th className="table-width-sm" aria-label="first-emptiness" />
```

Comment:
These aria-labels are important for screen readers so use the comments to give them descriptive names. If you are not certain how what they are for, please inspect the elements in your browser or ask us :)